src/ompl/base/samplers/InformedStateSampler.h

```diff
@@ -111,9 +111,12 @@ namespace ompl
             /** \brief The definition of a function pointer for querying the current solution cost. */
             typedef boost::function<Cost ()> GetCurrentCostFunc;
 
-            /** \brief Construct a sampler that only generates states with a heuristic solution estimate that is less than the cost of the current solution. Requires a function pointer to a method to query the cost of the current solution. */
+            /** \brief Construct a sampler that only generates states with a heuristic solution estimate that is less than the cost of the current solution using the default informed sampler for the current optimization objective. Requires a function pointer to a method to query the cost of the current solution. */
             InformedStateSampler(const ProblemDefinitionPtr &probDefn, unsigned int maxNumberCalls, const GetCurrentCostFunc &costFunc);
 
+            /** \brief Construct a sampler that only generates states with a heuristic solution estimate that is less than the cost of the current solution using the provided informed sampler. Requires a function pointer to a method to query the cost of the current solution. */
+            InformedStateSampler(const ProblemDefinitionPtr &probDefn, const GetCurrentCostFunc &costFunc, const InformedSamplerPtr &infSampler);
```

Comment:
A second version of the wrapper that allows for the explicit definition of the InformedSampler to use.